thirdeye/thirdeye-frontend/app/pods/rootcause/controller.js

```diff
@@ -451,6 +456,19 @@ export default Ember.Controller.extend({
       });
     },
 
+    /**
+     * Closure action passed into the legend component
+     * to handle the hover interactivity
+     * @param {String} urn
+     */
+    onLegendHover(urn) {
+      let id;
+      if (urn) {
+        id = urnToChartId(urn);
+      }
+      this.set('focusedId', id);
+    },
```

Comment:
maybe rootcause-chart is a better place to put this logic - it does all the translation between "rootcause entities" and "timeseries-chart series"